assets/tests/components/app.test.tsx

```diff
@@ -1,8 +1,125 @@
 import React from "react"
 import renderer from "react-test-renderer"
 import App from "../../src/components/app"
+import { Route, TimepointsByRouteId, Vehicle } from "../../src/skate"
 
-test("renders", () => {
-  const tree = renderer.create(<App />).toJSON()
+test("renders the empty state", () => {
```

Comment:
I like this new division between `App` and `AppStateWrapper` and how it enables these tests!